setup.py

```diff
@@ -10,4 +10,4 @@
         # If any package contains *.txt or *.rst files, include them:
         "": ["*.env"]
```

Comment:
Please remove from `setup.py` information that is currently in `setup.cfg` such as version, description, license, name.